.appveyor.ps1

```diff
@@ -0,0 +1,80 @@
+$ErrorActionPreference = "Stop"
+
+function run(){
+    param($command)
+    & $env:ComSpec /c ($command + ' 2>&1')
+}
+
+# consts
+$luatar = 'LuaJIT-2.0.5.zip'
```

Comment:
I'd rather not depend on a specific version in scripts but instead download the newest one.

Also, I hadn't considered yet if Lupa should ship with LuaJIT or plain Lua by default - it's probably ok since users get a generally faster system that way. Let's stick with LuaJIT here for now.